### ISSUE TITLE
prob question

### ISSUE THREAD

**Issue by Mulan337287** [Canjie-Luo/MORAN_v2#137]
```
torch.mean(preds0_prob[text_begin:text_begin+len(sim_preds0[j].split('$')[0]+'$')]).data[0] >\
                 torch.mean(preds1_prob[text_begin:text_begin+len(sim_preds1[j].split('$')[0]+'$')]).data[0]:
```
This code is to obtain the max probability, but it seems to torch.mean(softmax(preds0_prob)) should be calculated, not torch.mean(preds0_prob).

**Comment by Canjie-Luo** [comment]
Yes, you are right. But actually the result will be the same.
